UiPath.FreeRdpClient/UiPath.FreeRdpWrapper/FreeRdpWrapper.cpp

```diff
@@ -203,37 +231,35 @@ namespace FreeRdpClient
 		return 0;
 	}
 
-	instance_data* transport_start(rdpContext* context, LPCWSTR eventName)
+	instance_data* transport_start(rdpContext* context, ConnectOptions* rdpOptions)
 	{
-		instance_data* instanceData;
-		instanceData = (instance_data*)calloc(1, sizeof(instance_data));
+		instance_data* instanceData = new instance_data(context, rdpOptions);
 		if (!instanceData)
```

Comment:
Did we override `new` to not throw in case of memory issues or is this forgotten from struct times?

Reply:
I consider being outofmemory is acceptable to throw

Reply:
I agree. Therefore, the `if (!instanceData)` is misleading.